config/permissions.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+module.exports = function () {
+  return {
+    'cd-badges': {
+      'listBadges': [{
+        role: 'none'
+      }],
+
+      //  NOTE: Must be defined by visibility ?
```

Comment:
There is a couple of parameters on profile that defines visibility of fields on the public profile, that's somehting we should take into account maybe with a cp-visiblity ?